src/components/FieldHelp/index.js

```diff
@@ -63,13 +63,12 @@ class FieldHelp extends React.Component {
 }
 
 FieldHelp.propTypes = {
-  title: PropTypes.string,                                            // popover title
```

Comment:
Optional: I find this extra spaces helping with readability.

Reply:
Shorter lines looks better to me.
I think these spaces would be removed anyway within auto-reformating in most of the IDEs with default setting.